data/Acute_Toxicity_LD50/meta.yaml

```diff
@@ -0,0 +1,45 @@
+name: ld50_zhu
+description: Acute toxicity LD50 measures the most conservative dose that can lead
+  to lethal adverse effects. The higher the dose, the more lethal of a drug.
+targets:
+- id: acute_toxicity
+  description: Acute Toxicity LD50.
+  units: ld50
+  type: continuous
+  names:
+  - Acute Toxicity LD50
+  - ld50
+  - conservative dose that can lead to lethal adverse effects.
+  - Rat Acute Toxicity by Oral Exposure
+  - Toxicity
+identifiers:
+- id: SMILES
+  type: SMILES
+  description: SMILES
+- id: compound_name
+  type: Other
+  description: compound name
```

Comment:
is this a IUPAC-like name?

Reply:
@kjappelbaum May be I made a mistake. I will try to edit all PR again and remove compound id as well.

Reply:
yeah, Synonyms. some follow iupac name and some follow other names. I change it to Synonyms as I found most compound in PubChem Synonyms.